queries/vim/locals.scm

```diff
@@ -0,0 +1,10 @@
+[
+  (script_file)
+  (function_definition)
+] @scope
+
+(function_declaration name: (identifier) @definition.function)
+(function_declaration parameters: (parameters (identifier) @definition.parameter))
```

Comment:
Could this be made into one match?

Reply:
maybe, but it requires a query like `(parameters)?`, I prefer more explicit queries

Reply:
Indeed, it is better to keep it like this then.